inst/rmarkdown/templates/mdpi/skeleton/skeleton.Rmd

```diff
@@ -80,33 +137,197 @@ abbreviations:
     long: Three letter acronym
   - short: LD 
     long: linear dichroism
-output: rticles::mdpi_article
+bibliography: mybibfile.bib
+appendix: appendix.tex
+endnotes: false
+output: 
+  rticles::mdpi_article:
+    latex_engine: pdflatex #pdflatex or XeLaTeX allowed
 ---
 
 # Version
 
-This Rmd-skeleton uses the mdpi Latex template published 2019/02. 
+This Rmd-skeleton uses the mdpi Latex template published 2022-12-22. 
```

Comment:
I think they did another update since then. Probably small change compare to your work. 

Sorry for the delay on the PR. We should probably update to latest resource, shouldn't we ?